### ISSUE TITLE
Rule 'jest/no-standalone-expect' not configurable?

### ISSUE THREAD

**Issue by alexandros-megas** [jest-community/eslint-plugin-jest#623]
My project uses jest and puppeteer, and I have defined a custom test function called `journey`, such that I can write something like:

```javascript
describe('My app', () => {
  journey(
    'do something useful',
    signup(),
    completeProfile(),
    uploadAvatar(),
    async ({ page }) => {
      expect(page).toWhatever();
    },
  );
});
```

This `journey` function simply translates into an `it(...)` call that orchestrates running each of the steps in turn. I tried configuring the rule to recognize the `journey` function, but ESLint complains that the config is invalid (I have it in `overrides` because I only want this `journey` inside my end-to-end tests, not unit tests).

```json
  "overrides": [
    {
      "files": ["e2e/**/*.js"],
      "globals": {
        "page": true,
        "browser": true,
        "context": true,
        "jestPuppeteer": true
      },
      "rules": {
        "jest/no-standalone-expect": [
          "error",
          { "additionalTestBlockFunctions": ["journey"] }
        ]
      }
    }
  ]
```

And the error:

```
Oops! Something went wrong! :(

ESLint: 7.2.0

Error: .eslintrc#overrides[0]:
  Configuration for rule "jest/no-standalone-expect" is invalid:
  Value [{"additionalTestBlockFunctions":["journey"]}] should NOT have more than 0 items.

    at validateRuleOptions (C:\project\node_modules\eslint\lib\shared\config-validator.js:132:19)
    at C:\project\node_modules\eslint\lib\shared\config-validator.js:187:9
    at Array.forEach (<anonymous>)
    at validateRules (C:\project\node_modules\eslint\lib\shared\config-validator.js:184:30)
    at validateConfigArray (C:\project\node_modules\eslint\lib\shared\config-validator.js:312:9)
    at CascadingConfigArrayFactory._finalizeConfigArray (C:\project\node_modules\eslint\lib\cli-engine\cascading-config-array-factory.js:464:13)
    at CascadingConfigArrayFactory.getConfigArrayForFile (C:\project\node_modules\eslint\lib\cli-engine\cascading-config-array-factory.js:275:21)
    at CLIEngine.isPathIgnored (C:\project\node_modules\eslint\lib\cli-engine\cli-engine.js:953:18)
    at CLIEngine.executeOnText (C:\project\node_modules\eslint\lib\cli-engine\cli-engine.js:864:38)
    at ESLint.lintText (C:\project\node_modules\eslint\lib\eslint\eslint.js:554:23)
```

**Comment by G-Rath** [comment]
What version of `eslint-plugin-jest` are you using? This looks like a bug in `ESLint`.

**Comment by alexandros-megas** [comment]
I'm using `eslint-plugin-jest@23.13.2` and `eslint@7.2.0`

**Comment by G-Rath** [comment]
`additionalTestBlockFunctions` was added to `no-standalone-expect` in [`23.15.0`](https://github.com/jest-community/eslint-plugin-jest/blob/master/CHANGELOG.md#23150-2020-06-21) - updating the version of `eslint-plugin-jest` you're using should resolve this :)

**Comment by alexandros-megas** [comment]
That fixed it, thanks!
